From 2a08115f2c026232e2172db7f894d12306594d1d Mon Sep 17 00:00:00 2001
From: Markus Chou <chou.marcus@gmail.com>
Date: Sun, 7 Jul 2013 01:01:10 +0800
Subject: [PATCH] fix building on osx

    On Linux, they use 'BIND_4_COMPAT' and #include <arpa/nameser_compat.h> in <arpa/nameser.h>
    On OSX, if 'BIND_8_COMPAT' is #defined, <arpa/nameser_compat.h> is #included in <arpa/nameser.h>
---
 asn.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/asn.c b/asn.c
index f384f19..eac8d5e 100644
--- a/asn.c
+++ b/asn.c
@@ -21,7 +21,7 @@
 #include <stdlib.h>
 #include <sys/types.h>
 
-#ifndef __APPLE__
+#ifdef __APPLE__
 #define BIND_8_COMPAT
 #endif
 #include <arpa/nameser.h>
-- 
1.8.4

